tools/apiview/parsers/APIViewTreeStyleParserSchema/src/lib.ts

```diff
@@ -0,0 +1,20 @@
+import { createTypeSpecLibrary, paramMessage } from "@typespec/compiler";
+
+export const $lib = createTypeSpecLibrary({
+  name: "APIViewTreeStyleParserSchema",
+  // Define diagnostics for the library. This will provide a typed API to report diagnostic as well as a auto doc generation.
+  diagnostics: {
+    "banned-alternate-name": {
```

Comment:
I'm not sure why you'd be emitting diagnostics. The APIView TSP should just be the spec. Am I missing something?